tests/functional/setup.yml

```diff
@@ -32,9 +32,9 @@
       when: ansible_distribution == 'CentOS'
 
     - name: resize logical volume for root partition to fill remaining free space
-      command: lvresize -l +100%FREE /dev/atomicos/root
-      when: is_atomic
-
-    - name: increasing the size of the fs for root partition
-      command: xfs_growfs /
+      lvol:
+        lv: root
+        vg: atomicos
```

Comment:
looks like a mix of space/tabs isn't it ?

Reply:
L37 was an empty line while L36 gets two more spaces.